lib/node_modules/@stdlib/math/base/napi/unary/include/stdlib/math/base/napi/unary/d_d.h

```diff
@@ -48,17 +48,17 @@
 		napi_env env,                                                          \
 		napi_value exports                                                     \
 	) {                                                                        \
-		napi_value fcn;                                                        \
+		napi_value fcn_val;                                                   \
```

Comment:
```suggestion
		napi_value f;                                                   \
```

Thanks @Eternity0207 for pointing this out and fixing it! Much appreciated. Could you please make this change here and elsewhere as well?